cpplib/src/main/scala/scala/scalanative/cpp/ios/IOStream.scala

```diff
@@ -0,0 +1,53 @@
+package scala.scalanative.cpp.ios
+import scalanative.unsafe._
+import scala.scalanative.cpp._
+
+class IOStream(obj: NativeObject) extends CppObject(obj) {
```

Comment:
I don't feel like the weight of having a non-trivial model of C++ code is justified for thijs use case. Why not just use libc functions for the minimal I/O that we need in the core?

Reply:
You're right, after splitting any of this will not be needed, it will be just std_in/out/err and nothing else.